### PR TITLE
Chore: Put what's new and release notes URLs in package.json

### PR DESCRIPTION
**Pull request by aknuds1**
**What this PR does / why we need it**:
Put the what's new and release notes URLs in package.json, within a nested object called "grafana". This is so the build pipeline tool can auto detect these URLs, and doesn't have to hardcode them.

Also upgrade the build pipeline tool in .circleci/config.yml to a version that reads the URLs from package.json.

### CODE REVIEW

**Review by dprokop — APPROVED** [review]
👍

File: .circleci/config.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ commands:
       - run:
           name: "Install Grafana build pipeline tool"
           command: |
-            VERSION=0.2.12
+            VERSION=0.4.0
```
Comment (dprokop) [inline .circleci/config.yml:57]: Is this change intentional?

Comment (aknuds1) [reply .circleci/config.yml:57]: @dprokop Yes, I uploaded a preliminary version of the build pipeline tool that reads these URLs from Grafana's package.json.

File: scripts/build/publish.sh [hunk 2]
```diff
@@ -8,8 +8,8 @@ EXTRA_OPTS="$@"
 
 # Right now we hack this in into the publish script.
 # Eventually we might want to keep a list of all previous releases somewhere.
-_releaseNoteUrl="https://community.grafana.com/t/release-notes-v6-0-x/14010"
-_whatsNewUrl="http://docs.grafana.org/guides/whats-new-in-v6-0/"
+_releaseNoteUrl="https://community.grafana.com/t/release-notes-v7-0-x/29381"
```
Comment (marefr) [inline scripts/build/publish.sh:11]: Is this script still in use or should it be removed?

**Review by marefr — APPROVED** [review]
LGTM, but added acomment

File: scripts/build/publish.sh (hunk 2 above)
Comment (aknuds1) [reply scripts/build/publish.sh:11]: @marefr I think it's probably decommissioned, but haven't taken the effort of really checking.

**Review by daniellee — APPROVED** [review]
LGTM
